JS/Lesson14/tugba/script.js

```diff
@@ -69,11 +112,18 @@ console.log('------------------------');
 // 14. Use the `substring` method to extract the word "Databases" from string stringExercise14 based on two indices and print it to the console.
 // Question: What is the difference between `substring()` and `slice()`?
 const stringExercise14 = 'Databases are an essential component of many applications.';
+const extractedWord = stringExercise14.substring(0, 9);
+
+console.log("Extracted word:", extractedWord);
 
-console.log('------------------------');
 // 15. Remove extra spaces from the string stringExercise15 and print the result to the console.
 const stringExercise15 = "    Hello world    ";
+const trimmedString = stringExercise15.trim().replace(/\s+/g, ' ');
```

Comment:
I'd say using trim() should be enough in many cases. You can of course use regex but I'd say it's not necessary to combine them like this :)